Makefile

```diff
@@ -306,8 +306,17 @@ e2e-test:
 .PHONY: e2e-tests
 e2e-tests: # Run e2e tests
 e2e-tests:
+	# TEST_CASE=eth_to_azero::eth_to_weth make e2e-test # not working
```

Comment:
Please add comment either here or in some other prominent place, explaining that the tests are order-dependent

Reply:
done